apis/userapi/userapi.yaml

```diff
@@ -1721,6 +1796,135 @@ paths:
+        - name: X-Authenticated-User-Token
+          in: header
+          type: string
+          required: false
+          description: >-
+            if call with the auth token of same user will also give the user externalIds details in response.
+      responses:
+        '200':
+          description: 'OK ! Successful operation."Get User by ID Type" operation was successfully executed.'
+          schema:
+            $ref: '#/definitions/CreateUserSingleResp'
+        '400':
+          description: 'BAD REQUEST,"Get User by User ID Type" operation failed ! The possible reason for failure is that you may have missed providing input for a mandatory parameter.'
+          schema:
+            $ref: '#/definitions/UserErrorResponse'
+        '404':
+          description: 'Resource not found'
+          schema:
+            $ref: '#/definitions/UserErrorResponse'
+        '500':
+          description: >-
+            'INTERNAL SERVER ERROR. We track these errors automatically and try to set it right at the earliest. Try refreshing the page. If the problem persists contact us at info@sunbird.org.'
+          schema:
+            $ref: '#/definitions/UserErrorResponse'
+      security:
+        - bearer: []
+  /user/v1/exists/{idType}/{id}:
+    get:
+      description: >-
+        This API is used for searching a specific user, the user search is based on the unique ID type of the user
+          - The endpoint for **Get User by User ID** is `/user/v1/exists/{idType}/{id}`
+          - The fields marked with an asterisk (*) are mandatory. They cannot be null or empty.
+          - 'organisations' is to return organisation and location details.
+          - 'roles' is to return list of roles with name and ID details.
+
+      summary: Check User Exists by ID Type
+      tags:
+        - User APIs
+      operationId: check user exists
+      consumes:
+        - application/json
+      produces:
+        - application/json
+      parameters:
+        - in: path
+          name: idType
+          type: string
+          required: true
+          description: 'idType can only be either email, or phone'
+        - in: path
+          name: id
+          type: string
+          required: true
+          description: 'id can only be values of email or phone'
+        - name: ts
+          in: header
+          required: false
+          type: string
+          description: 'Timestamp at which given API request is sent.'
+        - name: X-msgid
+          in: header
+          required: false
+          type: string
+          description: 'This ID uniquely identifies a request if the same API is executed multiple times.'
+        - name: Authorization
+          in: header
+          required: true
+          type: string
+          description: 'Specify authorization key (format: Bearer api-key) received from administrator when performing given API request.'
+      responses:
+        '200':
```

Comment:
Add response for 400 bad request as well.

Reply:
done